packages/components/descriptions/src/description-item.ts

```diff
@@ -75,8 +75,10 @@ export const descriptionItemProps = buildProps({
     default: '',
   },
 })
+
+export const COMPONENT_NAME = 'ElDescriptionsItem'
```

Comment:
```suggestion
export const DescriptionItemComponentName = 'ElDescriptionsItem'
```
All variables in `description-item` will be exported. To avoid conflicts with other places and unclear meanings, it is better to change the name.
https://github.com/element-plus/element-plus/blob/7dbac0e1bc52a09b312bb15ad2b85320c127cfc1/packages/components/descriptions/index.ts#L19